### PR TITLE
fix in rhel8 service rex test

### PR DESCRIPTION
**Pull request by pondrejk**
postfix no longer installed by default on rhel8, playing with rsyslog instead. This service is enabled by default so adding a stop service step to make sure rex job takes effect

```
pytest tests/foreman/cli/test_remoteexecution.py -k test_positive_run_packages_and_services_job
================================================ test session starts ================================================
                                                   

tests/foreman/cli/test_remoteexecution.py .                                                                   [100%]


============================= 1 passed, 15 deselected, 5 warnings in 618.57s (0:10:18) ==============================
```

### CODE REVIEW

**Review by rplevka — APPROVED** [review]
Changes make sense and look sane to me.
ACK pending gh actions

File: tests/foreman/cli/test_remoteexecution.py [hunk 1]
```diff
@@ -825,11 +825,31 @@ def test_positive_run_packages_and_services_job(self, fixture_vmsetup, module_or
...
+        except AssertionError:
+            result = 'host output: {}'.format(
+                ' '.join(
+                    JobInvocation.get_output(
+                        {'id': invocation_command['id'], 'host': client.hostname}
+                    )
+                )
+            )
+            raise AssertionError(result)
+        result = client.execute(f"systemctl status {service}")
```
Comment (devendra104) [inline tests/foreman/cli/test_remoteexecution.py:849]:
```suggestion
        result = client.execute(f'systemctl status {service}')
```

File: tests/foreman/cli/test_remoteexecution.py [hunk 2]
```diff
@@ -825,11 +825,31 @@ def test_positive_run_packages_and_services_job(self, fixture_vmsetup, module_or
...
-        service = "postfix"
-        client.execute(
-            "sed -i 's/^inet_protocols.*/inet_protocols = ipv4/' /etc/postfix/main.cf",
+        # stop a service
+        service = "rsyslog"
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Service Action - Ansible Default',
+                'inputs': f'state=stopped, name={service}',
+                'search-query': f"name ~ {client.hostname}",
```
Comment (devendra104) [inline tests/foreman/cli/test_remoteexecution.py:834]:
```suggestion
                'search-query': f'name ~ {client.hostname}',
```

File: tests/foreman/cli/test_remoteexecution.py [hunk 3]
```diff
@@ -825,11 +825,31 @@ def test_positive_run_packages_and_services_job(self, fixture_vmsetup, module_or
         result = client.run(f'rpm -q {" ".join(packages)}')
         assert result.status == 0
 
-        # start a service
-        service = "postfix"
-        client.execute(
-            "sed -i 's/^inet_protocols.*/inet_protocols = ipv4/' /etc/postfix/main.cf",
+        # stop a service
+        service = "rsyslog"
```
Comment (devendra104) [inline tests/foreman/cli/test_remoteexecution.py:829]:
```suggestion
        service = 'rsyslog'
```

**Review by devendra104 — APPROVED** [review]
LGTM, left non-blocking comment.

**Pull request merged**